src/Tizen.NUI/src/public/Window.cs

```diff
@@ -209,6 +209,11 @@ public static Window Instance
         {
             get
             {
+                if (Application.Instance is ComponentApplication)
+                {
+                    Tizen.Log.Error("NUI", "You can't use 'Window.Intance' in ComponentApplication");
+                    throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
Why can not get the window of the ComponentApplication?
How to NUI API User get windows of ComponentApplication?